src/cli/backup/teams.go

```diff
@@ -0,0 +1,189 @@
+package backup
+
+import (
+	"github.com/alcionai/corso/src/cli/flags"
+	"github.com/alcionai/corso/src/cli/utils"
+	"github.com/alcionai/corso/src/pkg/path"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+// ------------------------------------------------------------------------------------------------
+// setup and globals
+// ------------------------------------------------------------------------------------------------
+
+const (
+	teamsServiceCommand                 = "team"
+	teamsServiceCommandCreateUseSuffix  = "--team <teamsName> | '" + flags.Wildcard + "'"
+	teamsServiceCommandDeleteUseSuffix  = "--backup <backupId>"
+	teamsServiceCommandDetailsUseSuffix = "--backup <backupId>"
+)
+
+const (
+	teamsServiceCommandCreateExamples = `# Backup all Teams data for Alice
+corso backup create team --team alice@example.com 
+
+# Backup only Teams contacts for Alice and Bob
+corso backup create team --team testTeams1,testTeams2 --data contacts
```

Comment:
Examples should contain data that reads like real world use cases.  `testTeams1` doesn't feel like a real usage.  

Also, this brings up the question: how are we identifying the team here?  What does the id structure look like?  Is there a human readable name that we can use to discover the team?

```suggestion
corso backup create team --team testTeams1,testTeams2 --data contacts
```

Reply:
Teams name do not seems to be unique. I will have to move it to teamID in that case.

Reply:
Teams have a webURL, similar to sharepoint sites.  Is that unique across teams?